modules/hyprlock/hm.nix

```
@@ -0,0 +1,22 @@
+{ config, lib, ... }:
+
+with config.lib.stylix;
+{
+  options.stylix.targets.hyprlock.enable =
+    mkEnableTarget "Hyprlock" true;
```

This fits on one line:

```suggestion
  options.stylix.targets.hyprlock.enable = mkEnableTarget "Hyprlock" true;
```